modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -3,10 +3,10 @@
 // * networking/ptp/configuring-ptp.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="nw-ptp-wpc-hardware-pins-reference_{context}"]
-= Intel Westport Channel E810 hardware configuration reference
+[id="nw-ptp-e810-hardware-pins-reference_{context}"]
+= Intel E810 NIC hardware configuration reference
 
-Use this information to understand how to use the link:https://github.com/openshift/linuxptp-daemon/blob/release-4.16/addons/intel/e810.go[Intel E810-XXVDA4T hardware plugin] to configure the E810 network interface as PTP grandmaster clock.
+Use this information to understand how to use the link:https://github.com/openshift/linuxptp-daemon/blob/release-4.16/addons/intel/e810.go[Intel E810 hardware plugin] to configure the E810 network interface as PTP grandmaster clock.
 Hardware pin configuration determines how the network interface interacts with other components and devices in the system.
 The E810-XXVDA4T NIC has four connectors for external 1PPS signals: `SMA1`, `SMA2`, `U.FL1`, and `U.FL2`.
```

Comment:
Here is another `E810-XXVDA4T` reference, I don't have a good suggestion how to handle it, maybe mention both WPC and LB models? Or `E810 (GNSS-enabled) NICs has...`

Reply:
Updated it as follows:
"The Intel E810 NIC has four connectors for external 1PPS signals: `SMA1`, `SMA2`, `U.FL1`, and `U.FL2`."

We have specified both the models in the intro "Configuring linuxptp services .... for dual E810 NICs". So just mentioning "Intel E810 NICs" should be fine.

Reply:
Alright, lgtm